include/deal.II/matrix_free/fe_evaluation.h

```diff
@@ -469,6 +469,17 @@ class FEEvaluationBase
   value_type
   get_laplacian(const unsigned int q_point) const;
 
+  /**
+   * Return the normal Hessian (i.e., the second derivative of the shape
+   * functions in the direction of the normal vector) of the finite element
```

Comment:
I do not have a better name for the function you propose here when considering compactness vs mathematical accuracy and intuition for the use. However, I would prefer if we can find a more mathematically precise description here. What do you think about
```suggestion
   * Return the second derivative along the normal direction $\partial_{n}^2 u_h
   * (i.e., the Hessian of the function $u_h$ contracted twice with the
   * direction of the normal vector) of the finite element
```

Reply:
Yes, this is more precise, I like it!